custom/icds_reports/ucr/data_sources/ag_care_monthly.json

```diff
@@ -0,0 +1,715 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-sql",
+    "icds-test",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-ag_care_monthly",
+    "display_name": "Cases - ag_care monthly",
+    "referenced_doc_type": "CommCareCase",
+    "description": "",
+    "base_item_expression": {
+      "type": "filter_items",
+      "items_expression": {
+        "expressions": [
+          0,
+          1,
+          2
+        ],
+        "type": "iterator",
+        "test": {}
+      },
+      "filter_expression": {
+        "type": "or",
+        "filters": [
+          {
+            "operator": "eq",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "root_doc",
+              "expression": {
+                "datatype": null,
+                "type": "property_name",
+                "property_name": "closed"
+              }
+            },
+            "property_value": false,
+            "comment": null
+          },
+          {
+            "operator": "gte",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "diff_days",
+              "from_date_expression": {
+                "type": "month_start_date",
+                "date_expression": {
+                  "date_expression": {
+                    "type": "icds_datetime_now"
+                  },
+                  "type": "add_months",
+                  "months_expression": {
+                    "type": "evaluator",
+                    "context_variables": {
+                      "iteration": {
+                        "type": "identity"
+                      }
+                    },
+                    "statement": "iteration - 2",
```

Comment:
based on how the other ones work, i think you want this to be `- 1`

Reply:
Sunaina/Akshita can you confirm I should make this change?